translations/Indonesian/general-rules.md

```diff
@@ -0,0 +1,26 @@
+# General Rules
+
+
+**Should translations be formal or informal?**
+Translations should be formal.
+
+**Which grammatical person should be used for translations (e.g. second person singular, first person plural)?**
+Use grammatical person as appropriate to the language. 
+
+**What verbs should be used (e.g. active, imperative, continuous)?**
+Imperative verbs should be used.
+
+**What types words or phrases should not be used (e.g. slang, regional)?**
+Slang or regional words and phrases should not be used.

```

Comment:
Any other types of words? Or specific words that shouldn't be used, e.g. because they're informal?